src/components/Header/components/MenuWrapper/components/LangSelect/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CURRENT_LANG, LANGUAGES } from 'appConstants';
+import { selectCurrLanguage } from 'modules/LoginPage/selectors';
+import { setCurrLang } from 'modules/LoginPage/loginPageReducer';
+import i18n from 'translation/resources';
+import { CommonSelectList } from 'components';
+
+type LangSelectProps = {
+  menuToggle?: string;
+};
+
+export const LangSelect: FC<LangSelectProps> = ({ menuToggle }) => {
+  const [displayLangList, setDisplayLangList] = useState(false);
+  const dispatch = useDispatch();
+  const currentLanguage = useSelector(selectCurrLanguage);
+
+  const onLangChange = (item: string) => {
+    setDisplayLangList(false);
+    localStorage.setItem(CURRENT_LANG, item);
```

Comment:
I'd advise you to create a redux middleware that will update local storage for some actions.

Reply:
Done